src/Components/Field/Field.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Field.css'
+
+export default ({
+    plantSeeds,
+    seedTypes,
+    field,
+    harvest
+}) =>
+    <div data-is="field" className={field.seedType !== null ? field.seedType.name : ""}>
+        
```

Comment:
Cette ligne vide ne sert pas a grand chose IMHO ;)

Reply:
si je met  "className={field.seedType !== null && field.seedType.name}", ça fonctionne, mais la console renvoie une erreur : 
Warning: Received `false` for a non-boolean attribute `className`.

If you want to write it to the DOM, pass a string instead: className="false" or className={value.toString()}.

If you used to conditionally omit it with className={condition && value}, pass className={condition ? value : undefined} instead.

Je propose donc de mettre undifined, comme ils ont envie... xD